### DISCUSSION TITLE
Sudo shouldn't force the mobile app as the default two-factor authentication provider

### DISCUSSION THREAD

**Discussion by sjdweb** [community/community#34708] [Other Feature Feedback, Questions, & Ideas]
When entering "sudo" mode (such as repo settings), Github now forces Github mobile as the suggested 2fa provider. There's no option to set a default provider, meaning if you use an app like 1Password you have extra steps every time. This is not a good user experience and led me to revoke the mobile app and uninstall it.

**Comment by sudoforge** [comment]
Absolutely agree with this, and I have removed the GitHub application from my mobile device(s) as well. I made another suggestion related to GitHub's implementation of MFA that I think you may agree with: https://github.com/orgs/community/discussions/35980

**Comment by jpttsn** [comment]
Thanks @sjdweb I too will try uninstalling the mobile app to remove this disruption. Pity.
